filters_test.go

```diff
@@ -346,6 +348,68 @@ func TestMatch(t *testing.T) {
 	}
 }
 
+func TestReadInput(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		Description string
```

Comment:
This doesn't match the look and feel of test cases in other tests.

Reply:
I agree. But I chose to follow the canon example of parallel table driven testing as demonstrated in [the golang wiki](https://github.com/golang/go/wiki/TableDrivenTests#parallel-testing).

I personally think it's better if the other tests followed _this_ format.

Reply:
Sorry, I should have been more specific: I meant that the field names are not exported, for example (and it's the same in the Go wiki you referenced).

What format were you referring to, then?

Reply:
Sorry, I got things mixed up. Fixed, samll letters are used as per the guide.